noun/retrieve.c

```diff
@@ -1501,7 +1506,17 @@ u3r_mug_string(const c3_c *a_c)
 c3_w
 u3r_mug_words(const c3_w* key_w, c3_w len_w)
 {
-  return u3r_mug_bytes((c3_y*)key_w, 4 * len_w);
+  c3_w siz_w = 0;
+  c3_w byt_w = 0;
+  c3_w wor_w;
+
+  while ( siz_w < len_w ) {
```

Comment:
might be better as a for loop, but whatever.

Reply:
I simplified it, still a while though.